assignment_6.py

```diff
@@ -0,0 +1,49 @@
+# Rearranging cars
+
+def rearrange_cars(cars,order):
+    '''

```

Comment:
According to Google Python Style Guide:

- It would be great to have a docstring explaining what a function does before describing its arguments and what it returns
- Aspects of a function should be documented in Args/Returns/Raises sections. Each section begins with a heading line, which ends with a colon. Sections should be indented two spaces, except for the heading.

Example:

    """Fetches rows from a Bigtable.

    Retrieves rows pertaining to the given keys from the Table instance
    represented by big_table.  Silly things may happen if
    other_silly_variable is not None.

    Args:
        big_table: An open Bigtable Table instance.
        keys: A sequence of strings representing the key of each table row
            to fetch.
        other_silly_variable: Another optional variable, that has a much
            longer name than the other args, and which does nothing.

    Returns:
        A dict mapping keys to the corresponding table row data
        fetched. Each row is represented as a tuple of strings. For
        example:

        {'Serak': ('Rigel VII', 'Preparer'),
         'Zim': ('Irk', 'Invader'),
         'Lrrr': ('Omicron Persei 8', 'Emperor')}

        If a key from the keys argument is missing from the dictionary,
        then that row was not found in the table.

    Raises:
        IOError: An error occurred accessing the bigtable.Table object.
    """
    pass

Could you please make the code more googley?:)